guide/payments/hide-sensitive-info.md

```diff
@@ -0,0 +1,107 @@
+
+It's more common for wallets to protect private keys but not much is done for other sensitive information like the balance, addresses and previous transactions but a few wallets like Bitcoin Core, Wasabi, Wallet of Satoshi, and others have made it work though. Below describes a pattern, and considerations for hiding and revealing sensitive information
+
+## Hiding and revealing information with the hide icon
+
+The hide icon / button, which is usually displayed within close reach of the balance itself, is used to quickly and easily hide and reveal wallet information by tapping or clicking on it.
+
+![hidden by eye icon](/assets/images/payments/hide-info-hidden-by-eye-icon.png)
+
+Pros
+
+- It is quite convenient to switch between revealed and hidden state
+
+Cons
+
+- Super easy for anyone else to reveal your information if they have access to your device.
+
+
+## Entering a pin to unveil information
+
+A problem that seemed to arise throughout each implementation was that is was as easy to reverse the hidden state as it was to enable it. This may be for convenience sake, however, if you are in a situation where you are concerned about unauthorized access by someone who has your device, once hidden perhaps you should be only able to unhide if a PIN or password has been entered. This could therefore reaffirm the identity of the wallet owner for extra security.
+
+![reveal by pin](/assets/images/payments/hide-info-reveal-by-pin.png)
+
+Pros
+- The risk of an unauthorized person revealing your information is minimal due to the pin required
+
+Cons
+
+- It may be annoying having to repeatedly put in your min when ever you want to reveal your information especially if you do so often
+
+  
+
+##  Information hidden by default
+
+Another solution would be to invoke the wallet’s hidden state as a default when the app is opened. What this means is that in the event that someone sees your wallet when you open it, they wouldn't be able to make out the information. The pre-hidden state can be unveiled  after a tap, PIN entry, or perhaps a short 5 second timer.
+
+![hidden by default](/assets/images/payments/hide-info-hidden-by-default.png)
+
+Pros
+
+- Users have some time to assess their environment before their info is displayed.
+
+Cons
+
+- The user may feel a bit frustrated having to wait for their information to be revealed especially in an urgent situation
+
+## Consider removing the hide toggle from the homepage
+
+Having the show/hide button right on the main screen make things quite obvious for someone who has access to your device to press unhide. A solution would be to move the hide toggle away from the home screen and into the app settings. This way, if someone has access to your device and opens the app, they may not immediately know how to reveal the hidden information as it is not as obvious as the previous solutions.
```

Comment:
Might be worth also including what information will be hidden as well.

Reply:
Alright, that's noted